### PR TITLE
Add a short JDBC user guide

### PR DESCRIPTION
**Pull request by HyukjinKwon**
Screen Shot 2021-04-12 at 5 04 07 PMScreen Shot 2021-04-12 at 5 05 00 PM

### CODE REVIEW

**Review by itholic — APPROVED** [review]
Thanks! LGTM

File: docs/source/user_guide/from_to_dbms.rst [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+    from pyspark.sql import SparkSession
+
+    (SparkSession.builder
+        .master("local")
+        .appName("SQLite JDBC")
+        .config(
+            "spark.jars",
+            "{}/sqlite-jdbc-3.34.0.jar".format(os.getcwd()))
+        .config(
+            "spark.driver.extraClassPath",
```
Comment (ueshin) [inline docs/source/user_guide/from_to_dbms.rst:72]: Do we need to set `spark.executor.extraClassPath` as well?

**Review by ueshin — APPROVED** [review]
Otherwise, LGTM, pending tests.

File: docs/source/user_guide/from_to_dbms.rst (hunk 1 above)
Comment (ueshin) [reply docs/source/user_guide/from_to_dbms.rst:72]: Maybe not:

> Extra classpath entries to prepend to the classpath of executors. This exists primarily for backwards-compatibility with older versions of Spark. Users typically should not need to set this option.

https://spark.apache.org/docs/latest/configuration.html#runtime-environment

Comment (ueshin) [reply docs/source/user_guide/from_to_dbms.rst:72]: But it might still be needed if the `older versions of Spark` includes `2.4`?

Comment (HyukjinKwon) [reply docs/source/user_guide/from_to_dbms.rst:72]: I think it's okay to just leave it ... I referred to https://spark.apache.org/docs/3.1.1/sql-data-sources-jdbc.html#content.

**Review by xinrong-databricks — APPROVED** [review]

File: docs/source/user_guide/from_to_dbms.rst [hunk 2]
```diff
@@ -0,0 +1,107 @@
+====================
+From/to other DBMSes
+====================
+.. currentmodule:: databricks.koalas
+
+
+The APIs interacting with other DBMSes in Koalas is slightly different from the ones in pandas
```
Comment (xinrong-databricks) [inline docs/source/user_guide/from_to_dbms.rst:7]: nit: are?

The APIs interacting with other DBMSes in Koalas **are** slightly ...

File: docs/source/user_guide/from_to_dbms.rst [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+    cur.execute(
+        '''CREATE TABLE stocks
+           (date text, trans text, symbol text, qty real, price real)''')
+    # Insert a row of data
+    cur.execute("INSERT INTO stocks VALUES ('2006-01-05','BUY','RHAT',100,35.14)")
+    # Save (commit) the changes
+    con.commit()
+    con.close()
+
+Koalas requires a JDBC driver to read so it requires the driver for your particular database on the Spark's classpath. For SQLite JDBC driver, you can download it, for example, as below:
```
Comment (xinrong-databricks) [inline docs/source/user_guide/from_to_dbms.rst:51]: nit: to be

... so it requires the driver for your particular database **to be** on the Spark's classpath.

**Comment by xinrong-databricks** [comment]
LGTM, thanks!

File: docs/source/user_guide/from_to_dbms.rst (hunk 2 above)
Comment (HyukjinKwon) [inline docs/source/user_guide/from_to_dbms.rst:7]:
```suggestion
The APIs interacting with other DBMSes in Koalas are slightly different from the ones in pandas
```

File: docs/source/user_guide/from_to_dbms.rst (hunk 3 above)
Comment (HyukjinKwon) [inline docs/source/user_guide/from_to_dbms.rst:51]:
```suggestion
Koalas requires a JDBC driver to read so it requires the driver for your particular database to be on the Spark's classpath. For SQLite JDBC driver, you can download it, for example, as below:
```

**Comment by ueshin** [comment]
Thanks! merging.

**Pull request merged**